recipes-tag/streamz_ext/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "streamz_ext" %}
-{% set version = "0.1.0" %}
-{% set sha256 = "110256c72aefb9229315d433dc4dd5599ee85486ff416f87433dc3da1042c549" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "e8054418130ecb28272181f626f0c65d7596be73462f97ffa2e584ee75720b49" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/xpdAcq/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://github.com/xpdAcq/{{ name }}/releases/download/{{ version }}/{{ version }}.tar.gz
```

Comment:
What's the difference between the `archive` and the `releases/download` here? Is it something new by GitHub?

Reply:
Oh, good to know, thanks!